pulsar-io/kafka-connect-adaptor/src/main/java/org/apache/pulsar/io/kafka/connect/KafkaConnectSource.java

```diff
@@ -188,15 +198,48 @@ public void close() {
         @Getter
         Optional<String> destinationTopic;
 
+        private final AvroData avroData;
+
+        private org.apache.pulsar.kafka.shade.avro.Schema keyAvroSchema;
```

Comment:
I don't think this needs to be a class variable. what method is using this variable except the constructor?